prepare-videochat.sh

```diff
@@ -338,7 +347,7 @@ fi
 if lsmod | grep v4l2loopback >/dev/null 2>/dev/null; then
     # module is already loaded, do nothing
     :
-else
+elif [ $CAPTURE_STREAM = v -o $CAPTURE_STREAM = av ]; then
```

Comment:
Sorry, I did not notice your wish to document CAPTURE_STREAM variable. Done now.